hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -417,7 +426,7 @@ public FSDataOutputStream createNonRecursive(final Path f, final FsPermission pe
           + f.getName() + " because parent folder does not exist.");
     }
 
-    return create(f, permission, overwrite, bufferSize, replication, blockSize, progress);
+    return createInternal(f, permission, overwrite, blockSize, true);
```

Comment:
Seems like these parameters were not used. Should we still keep them to reduce unnecessary diffs??
No, issues in removing as well.

Reply:
Taken.